### PR TITLE
Include referer in API request

### PR DESCRIPTION
**Pull request by RadLikeWhoa**
This PR fixes the issue I described in #40.

I had to upgrade the Google PHP client to version 2 as version 1 did not allow modifying the HTTP client. Fortunately there weren't a lot of changes that affected this package.

### CODE REVIEW

File: Classes/Controller/ConfigurationController.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function authenticateAction()
         if (!empty($code)) {
             $client->authenticate($code);
 
-            $this->tokenStorage->storeAccessToken($client->getAccessToken());
+            $this->tokenStorage->storeAccessToken(json_encode($client->getAccessToken()));
```
Comment (kdambekalns) [inline Classes/Controller/ConfigurationController.php]: Why is this needed?

File: Classes/Service/GoogleAnalytics.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ class GoogleAnalytics extends \Google_Service_Analytics
      */
     public function requireAuthentication()
     {
-        if ((string)$this->getClient()->getAccessToken() === '') {
```
Comment (kdambekalns) [inline Classes/Service/GoogleAnalytics.php]: Why is this needed?

**Comment by DrillSergeant** [comment]
Seems like we stuck in this MR. @RadLikeWhoa can you answer @kdambekalns questions so the merge can go on?
I am interested in a fixed version because our API keys are always referrer-protected and right now I cannot use the analytics-module.

**Comment by RadLikeWhoa** [comment]
@kdambekalns The change was due to a change in the new version of the API client (https://github.com/google/google-api-php-client/blob/master/UPGRADING.md#access-tokens-are-passed-around-as-arrays-instead-of-json-strings).

**Comment by DrillSergeant** [comment]
@RadLikeWhoa thanks for the fast reaction.
